tests/unit/components/Records/Search/Output/SearchLinkChips.spec.js

```diff
@@ -52,7 +52,22 @@ describe("SearchLinkChips.vue", function () {
         let chip = {label: 'orange', active: 'false'};
         wrapper.vm.updateSearchQuery(chip);
         expect($router.push).toHaveBeenCalledTimes(3);
+
+
     });
 
+    it("allows multiple fields of same tag type", () => {
```

Comment:
This can be improved by adding an extra expect($router.push).toHaveBeenCalledWith({....}) to check the input given to the push mock.